templates/main/index.html.ep

```diff
@@ -21,8 +21,6 @@
 
 % for my $groupresults (@$results) {
     % my $group = delete $groupresults->{_group};
-    <h2>
-        %= link_to $group->name => url_for('group_overview', groupid => $group->id)
-    </h2>
+    <h2><%= link_to $group->name => url_for('group_overview', groupid => $group->id) %></h2>
```

Comment:
I wonder why you changed this. I prefer the %= syntax over the %></ inline smileys